Template10 (Validation)/Validation/ValidatableViewModelBase.cs

```diff
@@ -9,16 +10,17 @@
 
 namespace Template10.Validation
 {
-    public abstract class ModelBase : BindableBase, IModel
+    public abstract class ValidatableViewModelBase : ViewModelBase, IModel
     {
-        public ModelBase()
+        public ValidatableViewModelBase()
```

Comment:
This is to be the base class of a Model, right? Not a View-model. It seems to me that we would call this `ValidatableModelBase` since it is the base for a model. If it were the base for a view-model we would call it `ValidatableViewModelBase`. But, it isn't. This is for models. Right?